plugins/plugin-maven/che-plugin-maven-ide/src/main/java/org/eclipse/che/plugin/maven/client/MavenExtension.java

```diff
@@ -65,17 +68,29 @@ public MavenExtension(PreSelectedProjectTypeManager preSelectedProjectManager,
     @Inject
     private void prepareActions(ActionManager actionManager,
                                 DependencyResolverAction dependencyResolverAction,
-                                GetEffectivePomAction getEffectivePomAction) {
+                                GetEffectivePomAction getEffectivePomAction,
+                                ReimportMavenDependenciesAction reimportMavenDependenciesAction) {
         // register actions
         actionManager.registerAction("getEffectivePom", getEffectivePomAction);
+        actionManager.registerAction("reimportMavenDependenciesAction", reimportMavenDependenciesAction);
 
         // add actions in main menu
-        DefaultActionGroup assistantGroup = (DefaultActionGroup)actionManager.getAction(IdeActions.GROUP_ASSISTANT);
+        DefaultActionGroup assistantGroup = (DefaultActionGroup)actionManager.getAction(GROUP_ASSISTANT);
         assistantGroup.add(getEffectivePomAction, Constraints.LAST);
 
+        // create maven context menu
+        DefaultActionGroup mavenContextMenuGroup = new DefaultActionGroup("Maven", true, actionManager);
```

Comment:
so it would mean that if someone want to get this group from another class, it will use "Maven" instead reusing a constant variable ?